src/main/scala/com/typesafe/sbt/packager/rpm/Keys.scala

```diff
@@ -60,4 +60,15 @@ trait RpmKeys {
   // Building
   val rpmLint = TaskKey[Unit]("rpm-lint", "Runs rpmlint program against the genreated RPM, if available.")
 
+  val rpmDaemonLogFile = SettingKey[String]("rpm-daemon-log-file", "Name of the log file generated by application daemon")
+
+  val rpmScriptReplacements = SettingKey[Seq[(String, String)]](
```

Comment:
I understand that `rpmScriptReplacements` is declared here because it is specific to RPM i.e. it isn't a script replacement destined for Linux.

If there was a key named just `scriptReplacements` then that'd be different...

Make sense @muuki88 ?